docs/changelog.rst

```diff
@@ -4,6 +4,19 @@
 Changes in nbformat
 =========================
 
+4.3
+===
+
+`4.3 on GitHub <https://github.com/jupyter/nbformat/milestone/7>`__
+
+- A new pluggable ``SignatureStore`` class allows specifying different ways to
+  record the signatures of trusted notebooks. The default is still an SQLite
```

Comment:
Interesting. I pronounce it 'ess-queue-ell-ite', so to me 'an' fits better. How do you pronounce it?